core/src/main/scala/filodb.core/zipkin/Zipkin.scala

```diff
@@ -137,8 +139,8 @@ class ZipkinReporter extends SpanReporter {
       OkHttpSender.newBuilder()
         .encoding(Encoding.JSON)
         .endpoint(url)
-        .maxRequests(64)
-        .messageMaxBytes(1024 * 128)
+        .maxRequests(if (maxRequests > 0) maxRequests else 128)
+        .messageMaxBytes(if (messageMaxBytes > 0) messageMaxBytes else 1024 * 256)
```

Comment:
Can we add the default to filodb-defaults.conf ?

Reply:
Even then we might need this check.. Because the value can be 0 ?

Reply:
Ah that's true.  Probably need to check regardless